Jenkinsfile

```diff
@@ -47,15 +49,16 @@ node {
 
     onlyOnMaster {
         stage('release') {
-            releaseApp(image: img)
+            releaseApp(image: image)
+            releaseApp(image: ws_only_image)
```

Comment:
Do we maybe want to try this first without releasing it?

Reply:
As we are going to be pushing a second image, we are going to need a new repo on docker to host it. Once that has been setup, I think we should be good to release it as part of the pipeline. The QA environment will get updated with the H image it has always used, and will not know about the new `ws_only_image`.

I will create us a new repo called: `hypothesis-ws`

Reply:
https://hub.docker.com/repository/docker/hypothesis/hypothesis-ws/